build.sbt

```diff
@@ -22,12 +22,6 @@ lazy val jawnSettings = Seq(
 lazy val jawnSettingsJVM = List(Test / fork := true)
 lazy val jawnSettingsJS = List(
   tlVersionIntroduced := List("2.12", "2.13", "3").map(_ -> "1.2.0").toMap,
-  scalaJSLinkerConfig ~= {
-    _.withSemantics(
-      _.withAsInstanceOfs(org.scalajs.linker.interface.CheckedBehavior.Unchecked)
-        .withArrayIndexOutOfBounds(org.scalajs.linker.interface.CheckedBehavior.Unchecked)
-    )
-  }
```

Comment:
Although in production mode these are undefined behaviors, in development mode Scala.js raises fatal errors to help detect these issues. As a workaround, we were previously disabling this to silence the fatal errors (my bad 😅 ).